main.go

```diff
@@ -369,10 +367,29 @@ backports to. For example:
 	}
 	c.username = m[2]
 
-	// Build GitHub client.
+	// Warn if legacy token is configured in git config.
+	if legacyToken, _ := capture("git", "config", "--get", "cockroach.githubToken"); legacyToken != "" {
```

Comment:
`cockroach.githubToken` is also used in `pkg/cmd/skip-test` in `cockroach`. So if someone follows the advice to remove it here, `skip-test` may start to not work.

Reply:
maybe we should fix it too? WDYT?